### PR TITLE
Minor fixes modal and audioplayer

### PR DESCRIPTION
**Pull request by alexander-daniel**

### CODE REVIEW

File: src/ts/views/modals/modal.tsx [hunk 1]
```diff
@@ -21,9 +21,7 @@ const Modal = (props: {
   return (
     <div className={wrapperClass}>
       <div className="modal">
-        <div className="close-button" onClick={onClose}>
-          {'X'}
-        </div>
+        <div className="icon-x close-button" onClick={onClose} />
```
Comment (Sundiata2) [inline src/ts/views/modals/modal.tsx]: is this 2 different icons or something?

File: viewer-app/root.tsx [hunk 2]
```diff
@@ -281,12 +283,12 @@ class Root extends React.Component<{}, {
         </ComponentSection>
 
         <ComponentSection title={'Audio Player'}>
-          {/* <AudioPlayer
-            audioURL={'http://files.platform.test/audio/1/1c31d054ed9c7420.mp3'}
+          <AudioPlayer
+            audioURL={'https://archive.org/download/78_burgundy-street-blues_george-lewis-his-new-orleans-music-george-lewis-elmer-talbe_gbia0029860b/_78_burgundy-street-blues_george-lewis-his-new-orleans-music-george-lewis-elmer-talbe_gbia0029860b_01_3.8_CT_EQ.mp3'}
```
Comment (Sundiata2) [inline viewer-app/root.tsx]: lol this is so long but not much we can do about that

Comment (alexander-daniel) [reply viewer-app/root.tsx]: yeah... *shrug*

File: src/ts/views/modals/modal.tsx (hunk 1 above)
Comment (Sundiata2) [reply src/ts/views/modals/modal.tsx]: ok word

**Comment by Sundiata2** [comment]
🚚

**Pull request merged**